README.md

```diff
@@ -1 +1,63 @@
-# Global-Summit
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TPPS Conference (Microverse captsone 1)
+
+> This website (purely fictional) is 
+- Homepage             
+![](images/capstone-desktop.png.png)
```

Comment:
- [ ] It looks like you have a typo here in the image path. this caused the image not to appear in your README file. Please remove the extra `.png` in this image path 😉